frameworks/kafka/tests/test_zookeeper_authz.py

```diff
@@ -92,7 +92,10 @@ def zookeeper_server(kerberos):
 
     try:
         sdk_install.uninstall(config.ZOOKEEPER_PACKAGE_NAME, config.ZOOKEEPER_SERVICE_NAME)
-        sdk_security.setup_security(config.ZOOKEEPER_SERVICE_NAME, zk_account, zk_secret)
+        service_account_info = sdk_security.setup_security(config.ZOOKEEPER_SERVICE_NAME,
+                                                           linux_user="nobody",
```

Comment:
Would it be better if we moved the `"nobody"` value to a constant somewhere?

Reply:
@6776bd7